standalone/opts.go

```diff
@@ -80,6 +80,13 @@ func ServeAsset(filename string, contents []byte) HandlerOption {
 	})
 }
 
+// WithExamples will add examples to the UI. The examples blob is expected to be valid json.
+func WithExamples(examples []byte) HandlerOption {
```

Comment:
I don't think `[]byte` is sufficiently type-safe. Requiring a blob is both unsafe but also makes the API much harder to use for a casual reader. Simply "valid json" isn't enough either.

It should be pretty easy to model the actual type here with a Go struct w/ "json" struct tags. For your case, of just loading it from a JSON file, you can unmarshal it into the Go type, which provides a nice start-up check that the file is valid.

Reply:
Will work on this next. I'll add a `standalone.ParseExamplesFile(examplesFilePath)`, and `standalone.ParseExamples(examplesBlob)` that output a go json model.